typings.d.ts

```diff
@@ -4,7 +4,18 @@
 // are not supported
 declare module '*';
 
-declare var page_params: object;
+interface PageParamsObject extends object{
+    realm_uri: string;
+    default_language: string;
+}
+declare var page_params: PageParamsObject;
 
 // since requestIdleCallback didn't make it into lib.dom.d.ts yet
 declare function requestIdleCallback(callback: Function, options?: object): void;
+
+interface MyWindow extends Window {
```

Comment:
Let's call this `ZulipWebWindow` or something that makes it clear it's zulip web app window.